pyproject.toml

```diff
@@ -82,6 +82,10 @@ docs = [
 roman = [
     "roman_datamodels>=0.17.1",
 ]
+strauss = [
+    "strauss@git+https://github.com/james-trayford/strauss@equal_loudness_normalisation",
+    "sounddevice"
```

Comment:
Hmm does installing from branch like this not pick up deps from https://github.com/james-trayford/strauss/blob/ddaca6b823f774305c6520ef9afd714c48cea7ac/setup.cfg#L21 ?

Also now you have `E   OSError: PortAudio library not found`